fuzzing/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -eu
+
+sed 's/use_all_warnings/\#use_all_warnings/g' -i $SRC/igraph/src/CMakeLists.txt
+mkdir build && cd build
+cmake ..
+make -j$(nproc)
```

Comment:
FYI, if the build machine has `ninja`, you can use `cmake -GNinja ..` followed by `ninja -$(nproc)`; it is much faster on my machine than `make`. Not sure whether it matters in the build env, though.

Reply:
Good to know. I suggest checking that out in a separate iteration on this as there might be other modifications required to build with Ninja. Let's merge this commit in first and then I will be happy to check it out.

Reply:
Ideally though, the next step should be integrating the fuzzers into igraphs build system. I tried the make build just now, and it definitely builds quickly enough on the OSS-fuzz platform.